CLI/1. User Guides/2. 登入.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 登入 TWCC CLI | en
+tags: Guide, CLI, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/twccheader-en %}
+{%hackmd @docsharedstyle/default %}
+
+# Log in to TWCC CLI
+
+After installation, please prepare **Project ID**, **API key (login certificate)**, and then log in to TWCC CLI to start.
+
+## Preparation
+
+### 1. Project ID
+
+Sign in to [TWCC Portal](https://www.twcc.ai/), and check the **Project ID** of the using project on the homepage
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_1a1bd7090cfd898384a84b61ef57b747.png)
+
+
+
+
+### 2. API key
+
+- Click **the user name** in the upper right corner, and then click **API Key**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_00e143d58b5cfee093c4c95d32dd2042.png)
+
+
+
+- Click <i class="fa fa-clone" aria-hidden="true"></i> copy button to get your API key
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_ec8357b81ff676799f942907115ab77d.png)
+
+
+
+:::danger
+{%hackmd @docsharedstyle/important-en %}
+ 
+To ensure security, please:
+- **Avoid using the master key to use services, and do not leak the master key out**.
+- Update your key regularly (delete and then create a new one).
+:::
+
+
+## Log in to TWCC CLI
+
+- Enter the following commands, and enter **TWCC API key** and **Project ID** to log in to TWCC CLI

```

Comment:
**TWCC API key** and **Project ID**  >

 your **TWCC API key** and **Project ID**

Reply:
to log in to TWCC CLI >

to sign in.